tests/integration/components/power-select/a11y-test.js

```diff
@@ -289,34 +289,6 @@ test('Multiple-select: The searchbox has type `search` and `aria-controls=<id-of
   assert.ok(/^ember-power-select-options-ember\d+$/.test(find('.ember-power-select-trigger-multiple-input').attributes['aria-controls'].value), 'The `aria-controls` points to the id of the listbox');
 });
 
-test('Single-select: The listbox has `aria-controls=<id-of-the-trigger>`', function(assert) {
```

Comment:
These tests were removed as UL's no longer have the aria-controls attribute. 

Per #806 "Remove aria-controls from ul elements inside the listbox. This is invalid. These lists do not actually control the trigger."